prometheus/exporter/docker_inspect.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
+import subprocess
+import json
+import sys
+
+def parseDockerInspect(jsonStr):
+    jsonObject = json.loads(jsonStr)
+    labels = []
+    envs = []
+    inspectMetrics = {}
+
+    for key in jsonObject[0]["Config"]["Labels"]:
+        labels.append("container_label_{0}=\"{1}\"".format(key.replace(".", "_"), jsonObject[0]["Config"]["Labels"][key]))
+
+    for env in jsonObject[0]["Config"]["Env"]:
+        envItem = env.split("=")
+        envs.append("container_env_{0}=\"{1}\"".format(envItem[0].replace(".", "_"), envItem[1]))
+
+    inspectMetrics = {"env": envs, "labels": labels}
+    return inspectMetrics
+    
+def inspect(argv):
```

Comment:
>argv [](start = 12, length = 4)

You should process argv in main and pass specific parameters to inspect such as inspect(containerId).